src/index.css

```diff
@@ -54,8 +54,9 @@ body {
   font-style: normal;
   font-weight: 500;
   font-size: 12px;
-  width: 50%;
+  max-width: 400px;
   margin: 0;
   justify-content: center;
   align-items: center;
 }
```

Comment:
```suggestion
  max-width: 400px;
  text-align: center;
}
```

Reply:
we don't (and probably won't) have anything else in this div so we don't really need the justiy-content and align-items stuff. just the text alignment